torch/optim/_fused/__init__.py

```diff
@@ -0,0 +1,18 @@
+"""
+:mod:`torch.optim._fused` is a package implementing various optimization algorithms.
```

Comment:
why do we need new _fused class instead of a `fused=True` arg to regular optimizer (which is how it's implemented anyway?)

Reply:
I mimicked `torch.optim._multi_tensor` ones. if you don't think we need this, I'll remove this `_fused` directory.

Reply:
`_multi_tensor` are no longer in use, they are accessible through regular ones with a constructor arg. Also, please update docstring for adam/adamw optimizers to describe the new flag.

Reply:
Understood, so it looks like they are there mainly for test_optim.py at the moment.

Reply:
`test_optim` should also be rewritten to use regular optimizers with `multi_tensor` kwarg, and not `_multi_tensor`.

Reply:
there's still some `optim._multi_tensor` uses in test_optim.py but I accessed this to some extent in https://github.com/pytorch/pytorch/pull/81705/commits/97c106a69c1f5dcb33aff33982b95e12ccbd89bb and removed `torch.optim._fused` in 87dc7b32fbccd6b84691a54704355d7c3ef95d07.

Reply:
https://github.com/pytorch/pytorch/pull/83549 can be a follow-up